src/Components/Users/UserAdd.tsx

```diff
@@ -709,10 +709,24 @@ export const UserAdd = (props: UserProps) => {
                               <i className="fas fa-circle-dot" /> checking...
                             </span>
                           ) : (
-                            validateRule(
-                              usernameExists !== userExistsEnums.exists,
-                              "Username is available"
-                            )
+                            <>
+                              " "
```

Comment:
```suggestion

```
This seems unnecessary.
![image](https://user-images.githubusercontent.com/3626859/204769836-dee335d2-bceb-4add-b97d-d6addeb44998.png)